public/app/features/alerting/unified/components/contact-points/ContactPoint.tsx

```diff
@@ -41,7 +42,7 @@ export const ContactPoint = ({ disabled = false, contactPoint }: ContactPointPro
       <Stack direction="column" gap={0}>
         <ContactPointHeader
           contactPoint={contactPoint}
-          disabled={disabled}
+          disabled={disabled || isLoading(deleteRequestState)}
```

Comment:
This `disabled` prop is not used , I think we could remove it

Reply:
Yep it seems like this got refactored so we don't use this anymore